AmericanMedicalAssociation/ama-style-guide-2
# EWL-9032: Adds annual/interim module mock ups to styleguide.

## Ticket(s)
**Do not** submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Github Issue**
- n/a

**Jira Ticket**
- [EWL-9032: Home Page Annual/Interim Module Styleguide](https://issues.ama-assn.org/browse/EWL-9032)

## Description
Adds mockups for new homepage annual/interim meetings module into the style guide.


## To Test
- Open styleguide (Run gulp serve in styleguide folder)
- Under Organisms menu open Annual Interim Meeting block
- Confirm both blocks Annual Interim Meeting Block and Annual Interim Meeting Block Active match the following Zeplins:
Desktop: https://zpl.io/bPyBRpm
Tablet: https://zpl.io/beyyrQY
Mobile: https://zpl.io/boAJlQM

## Visual Regressions
- N/A

## Relevant Screenshots/GIFs
![Screen Shot 2021-09-02 at 3 39 20 PM](https://user-images.githubusercontent.com/67962801/131913050-219a1f5e-3bb4-4995-898d-b84a3c47600d.png)
![Screen Shot 2021-09-02 at 3 39 29 PM](https://user-images.githubusercontent.com/67962801/131913054-ca633627-8aac-44f0-8ea3-256ff9d1d46b.png)


## Remaining Tasks
- N/A

## Additional Notes
- N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
